src/main/java/it/gov/pagopa/pu/pagopapayments/connector/PagoPaApiClient.java

```diff
@@ -0,0 +1,48 @@
+package it.gov.pagopa.pu.pagopapayments.connector;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.gov.pagopa.pu.pagopapayments.dto.PaTaxonomyDTO;
+import it.gov.pagopa.pu.pagopapayments.exception.ApplicationException;
+import it.gov.pagopa.pu.pagopapayments.util.RestUtil;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+
+@Service
+@Slf4j
+public class PagoPaApiClient {
+
+  private final RestTemplate restTemplate;
+  private final ObjectMapper objectMapper;
+  private final String baseUrlGithub;
+
+  public PagoPaApiClient(
+    @Value("${rest.pagopa-platform.base-url}") String baseUrlGithub,
+    RestTemplateBuilder restTemplateBuilder,
+    ObjectMapper objectMapper) {
+    this.restTemplate = restTemplateBuilder.build();
+    this.baseUrlGithub = baseUrlGithub;
```

Comment:
just call it baseUrl
```suggestion
    @Value("${rest.pagopa-platform.base-url}") String baseUrl,
    RestTemplateBuilder restTemplateBuilder,
    ObjectMapper objectMapper) {
    this.restTemplate = restTemplateBuilder.build();
    this.baseUrl = baseUrl;
```